.php_cs.dist

```diff
@@ -8,8 +8,7 @@ return PhpCsFixer\Config::create()
     ->setRules([
         '@Symfony' => true,
         '@Symfony:risky' => true,
-        '@PHPUnit48Migration:risky' => true,
-        'php_unit_no_expectation_annotation' => false, // part of `PHPUnitXYMigration:risky` ruleset, to be enabled when PHPUnit 4.x support will be dropped, as we don't want to rewrite exceptions handling twice
+        '@PHPUnit55Migration:risky' => true,
```

Comment:
Can be `PHPUnit75Migration:risky`

`php_unit_dedicate_assert` should be configured to with `target = 3.5` as all methods are not (yet) pollyfiled (ie `assertDirectoryExist`) in phpunit 4